src/web/components/ArticleTitle.stories.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { css } from 'emotion';
+
+import { ArticleTitle } from './ArticleTitle';
+
+const Container = ({ children }: { children: JSX.Element | JSX.Element[] }) => (
+    <div
+        className={css`
+            width: 620px;
+            padding: 20px;
+        `}
+    >
+        {children}
+    </div>
+);
+const CAPI = {
+    tags: [],
+    guardianBaseURL: 'https://theguardian.com',
+    inLeftCol: true,
+    fallbackToSection: true,
+    frontendAssetsFullURL: 'http://localhost:9000/assets/',
+};
+const brexitCAPI = {
+    ...CAPI,
+    ...{
+        sectionLabel: 'Brexit',
+        sectionUrl: '/brexit',
+        badge: {
+            seriesTag: 'politics/series/brexit-how-it-came-to-this',
+            imageUrl: 'images/badges/EUReferendumBadge.svg',
+        },
+    },
+};
+
+const beyondTheBladeCAPI = {
+    ...CAPI,
+    ...{
+        sectionLabel: 'Beyond the blade',
+        sectionUrl: '/beyond-the-blade',
+        badge: {
+            seriesTag: 'membership/series/beyond-the-blade',
+            imageUrl: 'images/badges/beyondthebladebadge.svg',
+        },
+    },
+};
+
+/* tslint:disable */
```

Comment:
I dnt think we need this? I seem to not get an error without it

Reply:
This rule might have changed recently, if so, we can remove these in a new PR